### DISCUSSION TITLE
Better restart button text after updating

### DISCUSSION THREAD

**Discussion by airone01** [ppy/osu#27542] [Ideas] [answered]
When the game auto updates, a notification is spawned to indicate progress. After the download is complete, the notification turns green with a text for a game restart. When clicking this notification, the modal for when closing the game spawn.
It might just be me but I got confused when betting this modal. The text hints at "quitting" the game, even though it restarts by itself, which made me think I clicked on the wrong button.
The modal when spawned from this notification is already different because clicking "get me out" instantly restarts the game, instead of having to long-press it, so I believe changing the text to hint for a restart (and change the icon in the button as well) would be doable.

P.S: Still learning English, feel free to point out any error, I'm always learning 👍

**Comment by Joehuu** [comment, marked as answer]
The quit dialog will be removed when restarting in https://github.com/ppy/osu/pull/26558.
